src/patternfly/components/Dropdown/styles.scss

```diff
@@ -82,15 +82,15 @@
   &:hover,
   &.pf-m-hover {
     &::before {
-      border-color: var(--pf-c-dropdown__toggle--hover--BorderColor);
       border-width: var(--pf-c-dropdown__toggle--hover--BorderWidth);
```

Comment:
When I compare the pf-m-plain variant of button to dropdown, the button never shows a border, but the dropdown shows a border on hover and expanded state. The expanded state makes sense, but are the hover states meant to be similar?

Reply:
thanks Jenn. Ya originally the plain dropdown had a hover a blue border I changed it to the pf-black-600. I do think that the plain button and plain dropdown should align for hover. I'm going to check in with @mceledonia on this one.